bindings/python/iota_sdk/types/address.py

```diff
@@ -91,3 +79,31 @@ class AddressWithUnspentOutputs():
     key_index: int
     internal: bool
     output_ids: bool
+
+
+def address_from_dict(dict: Dict[str, Any]) -> Union[Ed25519Address, AccountAddress, NFTAddress]:
```

Comment:
Maybe we just call this something like deserialize?

Reply:
@Thoralf-M Are we addressing this ?